### DISCUSSION TITLE
i want to clear my previous image optimization usage for new one

### DISCUSSION THREAD

**Discussion by MikeyA-yo** [vercel/next.js#60399] [Help]
### Summary

 i deployed a nextjs project that has image optimization unknowingly  to me there was a limit, and this project isn't even important now i need image optimization for new important ones and i can't get it.

**Comment by Hemistone** [comment]
Any Updates? 

I'm also in the same situation. 

I want to turn off already optimized images, but in the docs the only thing I could do was adding 'unoptimized' in each <Image/> Component or in next.config.js for entire Images. 

I think by this process, I will not get further more Optimized Image Counts, 
but for now I want to disable the past optimized images and decline my 'optimized image counts'.

**Reply by MikeyA-yo** [reply]
no update
